pandas/core/indexes/period.py

```diff
@@ -133,29 +133,43 @@ class PeriodIndex(DatelikeOps, DatetimeIndexOpsMixin, Int64Index):
     Parameters
     ----------
     data : array-like (1-dimensional), optional
-        Optional period-like data to construct index with
+        Optional period-like data to construct index with.
     copy : bool
-        Make a copy of input ndarray
+        Make a copy of input ndarray.
     freq : string or period object, optional
-        One of pandas period strings or corresponding objects
+        One of pandas period strings or corresponding objects.
```

Comment:
I think we have a reference somewhere to the offsets page:http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases, can you add

Reply:
:ref:`offset aliases <timeseries.offset_aliases>`

Reply:
So I was looking at that but some of the strings do not work. For instance `BQS` is not a valid frequenct. Also the offset alias does not mention anything about strides which frequency also supports. Is this documented anywhere?

Reply:
Periods are time *spans* so things like month-start vs. month-end don't make sense. It's the etnire month.

I'm not sure if that's explained anywhere in http://pandas.pydata.org/pandas-docs/stable/timeseries.html. If not it'd be good to add.

What do you mean by strides?